CacheMultimap.adoc

```diff
@@ -46,33 +45,52 @@ public interface CacheMultimap<K, V> {
 
 The underlying implementation will wrap a normal Cache. Some of the suggestions :
 
-== Core Module
 
-[source,java]
-----
-public interface EmbeddedCacheManager {
- ...
- 
- <K, V> CacheMultimap<K, V> getCacheMultimap(String cacheName, boolean createIfAbsent);
-}
+Apparently there is a problem with functional commands.
+They don't really work efficiently over Hot Rod (does get/replace in a loop).
+We would need to add some more handling in the protocol to allow for only partial replication
+of values and only 1 remote call.
 
-----
 
-== Commons Module
+== Embedded Multimap
 
-[source, java]
-----
-public class CacheMultimapFactory {
+A new maven module is created.
 
-   static <K, V> CacheMultimap<K, V> CacheMultimapFactory.get(Cache<K,Object> c) {
-      ...
-   }
-}
-----
+To create a multimap cache
+
+```java
+      EmbeddedCacheManager cm = ...
+      MultimapCacheManager multimapCacheManager = EmbeddedMultimapCacheManagerFactory.from(cm);
```

Comment:
Hmm, ok. The difference is that there is single cache providing services to all counters, while there's a 1:1 relation between caches and multimaps. Will you prefix the cache names for multimap operations, or do you plan to openly admit that the cache can be accessed using the same name?